chain-core/src/init/config.rs

```diff
@@ -238,6 +238,10 @@ impl InitConfig {
             return Err(DistributionError::InvalidPunishmentConfiguration);
         }
 
+        if self.network_params.jailing_config.jail_duration > self.network_params.unbonding_period {
```

Comment:
I still don't understand this constraint, I think unbonding is already forbidden during `jail_duration`.

Reply:
it's due to the fact that evidence that triggers "jailing" may come / be detected after a valid unbond tx

Reply:
> it's due to the fact that evidence that triggers "jailing" may come / be detected after a valid unbond tx

So when that happens, after `max_evidence_age` and `slash_wait_period`, we needs to slash both bonded and unbonded coins?
I think this scenario leads to such constraint?:
```
unbondindg_period > max_evidence_age + slash_wait_period
```
To make sure unbonded coins are not withdrawn before slashing.

Reply:
currently, `jail_duration > slash_wait_period` -- in TM 0.32, (maximum) evidence age is in "blocks" which will be hard to enforce. In (recently released) TM 0.33, there's both evidence age in seconds and in blocks.
I'm not yet sure if that's the exact constraint to be added, but it should be re-visited -- I opened this issue: https://github.com/crypto-com/chain/issues/1016